toolkit/tools/pkg/imagecustomizerlib/customizeverity.go

```diff
@@ -46,7 +46,7 @@ func enableVerityPartition(buildDir string, verity *imagecustomizerapi.Verity, i
 	return true, nil
 }
 
-func addDracutModule(dracutModuleName string, dracutDriverName string, imageChroot *safechroot.Chroot) error {
+func addDracutModuleAndDriver(dracutModuleName string, dracutDriverName string, imageChroot *safechroot.Chroot) error {
```

Comment:
It might be goodness to move these 3 functions into their own file. e.g. `dracututils.go`.